src/actions/auth.js

```diff
@@ -33,64 +36,77 @@ export const logout = () => (dispatch, getState, { history }) => {
   history.push('/');
 };
 
+// Wrap FB SDK login as promise
+const FBSDKLogin = async FB => {
```

Comment:
Optional: 回傳 Promise 的時候可以免 async
```suggestion
const FBSDKLogin = FB => {
```